vulnerabilities/importers/apache_httpd.py

```diff
@@ -13,43 +13,48 @@
 import requests
 from bs4 import BeautifulSoup
 from packageurl import PackageURL
-from univers.version_range import VersionRange
+from univers.version_constraint import VersionConstraint
+from univers.version_range import GenericVersionRange
 from univers.versions import SemverVersion
 
 from vulnerabilities.importer import AdvisoryData
+from vulnerabilities.importer import AffectedPackage
 from vulnerabilities.importer import Importer
 from vulnerabilities.importer import Reference
 from vulnerabilities.importer import VulnerabilitySeverity
 from vulnerabilities.package_managers import GitHubTagsAPI
 from vulnerabilities.severity_systems import APACHE_HTTPD
-from vulnerabilities.utils import nearest_patched_package
+
+KNOWN_RANGE_EXPRESSIONS = {"<=", ">=", "?=", "!<", "="}
 
 
 class ApacheHTTPDImporter(Importer):
 
     base_url = "https://httpd.apache.org/security/json/"
-
-    def set_api(self):
-        self.version_api = GitHubTagsAPI()
-        asyncio.run(self.version_api.load_api(["apache/httpd"]))
-        self.version_api.cache["apache/httpd"] = set(
-            filter(
-                lambda version: version.value not in ignore_tags,
-                self.version_api.cache["apache/httpd"],
-            )
-        )
-
-    def updated_advisories(self):
+    spdx_license_expression = "Apache-2.0"
+    license_url = "https://www.apache.org/licenses/"
+
+    # For now, don't use the GH API
+    # def set_api(self):
+    #     self.version_api = GitHubTagsAPI()
+    #     asyncio.run(self.version_api.load_api(["apache/httpd"]))
+    #     self.version_api.cache["apache/httpd"] = set(
+    #         filter(
+    #             lambda version: version.value not in ignore_tags,
+    #             self.version_api.cache["apache/httpd"],
+    #         )
+    #     )
+
+    def advisory_data(self):
         links = fetch_links(self.base_url)
-        self.set_api()
-        advisories = []
+        # For now, don't use the GH API
+        # self.set_api()
```

Comment:
Do not keep dead commented out code around

Reply:
Done.